library/src/test/java/com/datatorrent/lib/formatter/JsonFormatterTest.java

```diff
@@ -198,15 +199,25 @@ public void testJSONToPOJONoFieldPOJO()
   @Test
   public void testApplication() throws IOException, Exception
   {
-    try {
-      LocalMode lma = LocalMode.newInstance();
-      Configuration conf = new Configuration(false);
-      lma.prepareDAG(new JsonFormatterApplication(), conf);
-      LocalMode.Controller lc = lma.getController();
-      lc.run(2000);// runs for 2 seconds and quits
-    } catch (ConstraintViolationException e) {
-      Assert.fail("constraint violations: " + e.getConstraintViolations());
-    }
+    LocalMode lma = LocalMode.newInstance();
+    Configuration conf = new Configuration(false);
```

Comment:
@shubham-pathak22 Expectation is as follows:
1. Collect the tuples from the operator under test.
2. Compare it against expected results.
3. Assert if results are matching.
4. localmode execution should wait untill output results are available instead of some arbitrary time. 

You may refer to https://github.com/apache/apex-malhar/blob/master/library/src/test/java/org/apache/apex/malhar/lib/fs/FSRecordReaderTest.java#L90-L102 for reference.